app/notifications/twilio_sender.rb

```
@@ -10,10 +10,13 @@ class TwilioSender
   @@client = Twilio::REST::Client.new(@@account_sid, @@auth_token)
 
   def self.createMessage(message_text:, to_number:)
-    @@client.messages.create(
-      from: @@from,
-      to: "+1#{to_number}",
-      body: message_text
-    )
+    begin
+      @@client.messages.create(
+        from: @@from,
+        to: "+1#{to_number}",
+        body: message_text
+      )
+    rescue Twilio::REST::TwilioError => e
+      Rails.logger.error e.message
```

I think we need to re-raise this error. Otherwise, the failed jobs code won't be triggered. Also, errors from here will be caught here: https://github.com/ctoec/care-4-kids-auto-notifier/blob/3dbacff8dccb0aebc52cd4410d7709eba8ba6b7e/app/jobs/notification_send_job.rb#L19-L21